webpackConfig/build-min.js

```diff
@@ -0,0 +1,17 @@
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const webpackMerge = require("webpack-merge");
```

Comment:
is this to do deep merging of objects? I don't love the idea of pulling in another random dependency for this